readme.md

```diff
@@ -106,6 +113,35 @@ Extension of the config file.
 
 You would usually not need this, but could be useful if you want to interact with a file with a custom file extension that can be associated with your app. These might be simple save/export/preference files that are intended to be shareable or saved outside of the app.
 
+#### migrations
+
+ type: `object`<br>
+ Default: `undefined`
+
+ Migrations to be run between versions.
+
+ Useful for transitioning conf changes between application versions. Ex:
+ ```js
+ // version is set to 0.0.1, the current app version is 2.0.8, old is set to 1
+ const store = new Conf({
+ 	migrations: {
+ 		'0.0.0': store => {
```

Comment:
This is a bad example, as there are no versions before this, it's not actually a migration.